governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -0,0 +1,411 @@
+          ) {
+            proposalIxs.push(parsedInstruction)
+          }
+        }
+        setProposalInstructions(proposalIxs)
+      }
+    }
+
+    fetchProposalInstructions()
+  }, [proposal, squads, cluster])
+
+  return proposalInstructions ? (
+    <div className="grid grid-cols-3 gap-4">
+      <div className="col-span-3 my-2 space-y-4 bg-[#1E1B2F] p-4 lg:col-span-2">
+        <h4 className="h4">Info</h4>
+        <hr className="border-gray-700" />
+        <div className="flex justify-between">
+          <div>Proposal Pubkey</div>
+          <div>{proposal.publicKey.toBase58()}</div>
+        </div>
+        <div className="flex justify-between">
+          <div>Creator</div>
+          <div>{proposal.creator.toBase58()}</div>
+        </div>
+        <div className="flex justify-between">
+          <div>Multisig</div>
+          <div>{proposal.ms.toBase58()}</div>
+        </div>
+      </div>
+      <div className="col-span-3 my-2 space-y-4 bg-[#1E1B2F] p-4 lg:col-span-1">
+        <h4 className="h4 mb-4">Results</h4>
+        <hr className="border-gray-700" />
+        <div className="grid grid-cols-3 justify-center gap-4 pt-5 text-center align-middle">
+          <div>
+            <div className="font-bold">Confirmed</div>
+            <div className="text-lg">{proposal.approved.length}</div>
+          </div>
+          <div>
+            <div className="font-bold">Cancelled</div>
+            <div className="text-lg">{proposal.cancelled.length}</div>
+          </div>
+          <div>
+            <div className="font-bold">Threshold</div>
+            <div className="text-lg">
+              {multisig.threshold}/{multisig.keys.length}
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="col-span-3 my-2 space-y-4 bg-[#1E1B2F] p-4">
+        <h4 className="h4">Instructions</h4>
+        <hr className="border-gray-700" />
+        {proposalInstructions?.map((instruction, index) => (
+          <>
+            <div key={index} className="flex justify-between">
+              <div>Instruction</div>
+              <div>{instruction.name}</div>
+            </div>
+            <div key={index} className="grid grid-cols-4 justify-between">
+              <div>Arguments</div>
+              {Object.keys(instruction.args).length > 0 ? (
+                <div className="col-span-4 mt-2 bg-darkGray2 p-4 lg:col-span-3 lg:mt-0">
+                  <div className="base16 flex justify-between pt-2 pb-6 font-semibold opacity-60">
+                    <div>Key</div>
+                    <div>Value</div>
+                  </div>
+                  {Object.keys(instruction.args).map((key, index) => (
+                    <div
+                      key={index}
+                      className="flex justify-between border-t border-beige-300 py-3"
+                    >
+                      <div>{key}</div>
+                      <div>{instruction.args[key]}</div>
```

Comment:
This line fails when `instruction.args[key]` is a pubkey

Reply:
https://xc-admin-frontend-r30ihde1s-pyth-web.vercel.app/?tab=proposals&cluster=devnet&proposal=6149HZRfyeRCJ6qQ2yzmcQQ3zfnooRxoStX8ksAPhAKW

Reply:
good catch, fixed